vk_video_encoder/libs/VkVideoEncoder/VkEncoderDpbAV1.cpp

```diff
@@ -255,7 +255,7 @@ int32_t VkEncDpbAV1::GetRefFrameDpbId(StdVideoAV1ReferenceName refName)
 
             int32_t refBufId = m_refBufIdMap[refName];
 
-            if ((refBufId >= 0) && (refBufId < STD_VIDEO_AV1_NUM_REF_FRAMES)) {
+            if ((refBufId >= 0) && (refBufId < (int32_t)STD_VIDEO_AV1_NUM_REF_FRAMES)) {
```

Comment:
Just to be more C++ aware

```suggestion
            if ((refBufId >= 0) && (refBufId < static_cast<int32_t>(STD_VIDEO_AV1_NUM_REF_FRAMES))) {
```